### PR TITLE
Added the C++ implementation

### PR DESCRIPTION
**Pull request by shubhangi013**
Added the C++ implementation of naive searching algorithm

### CODE REVIEW

File: c++/naive_pattern_searching.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	for (int i = 0; i <= n - m; i++) {
+		int j;
+
+		for (j = 0; j < m; j++)
+			if (string[i + j] != pattern[j])
+				break;
+        if (j == m)
+        //returning the index where "AN" occurs in "I AM SHUBHANGI. AN AVID READER WHO LIKES TO LISTEN TO MUSIC."
+			cout << "'AN' occurs in 'I AM SHUBHANGI. AN AVID READER WHO LIKES TO LISTEN TO MUSIC.' at :  "
+				 << i << endl;
```
Comment (king-11) [inline c++/naive_pattern_searching.cpp:18]: Would like to keep this dynamic

File: c++/naive_pattern_searching.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	}
+}
+
+
+int main()
+{
+	char string[] = "I AM SHUBHANGI. AN AVID READER WHO LIKES TO LISTEN TO MUSIC.";
+	char pattern[] = "AN";
```
Comment (king-11) [inline c++/naive_pattern_searching.cpp:26]: Take both `pattern` and `search_string` as input change the variable name too

File: c++/naive_pattern_searching.cpp [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#include <bits/stdc++.h>
```
Comment (king-11) [inline c++/naive_pattern_searching.cpp:1]: Try using the libraries that you need not the complete set of tools

File: c++/naive_pattern_searching.cpp [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+using namespace std;
+
+void patternSearch(char* pattern, char* string)
```
Comment (king-11) [inline c++/naive_pattern_searching.cpp:4]: - I would the like it to return the location of match not a void.
- secondly move the function definition below main and keep function declaration only above it just for clean code purposes

**Review by king-11 — CHANGES_REQUESTED** [review]
Algorithm LGTM 👍 just some cleaning is needed

File: c++/naive_pattern_searching.cpp (hunk 2 above)
Comment (shubhangi013) [reply c++/naive_pattern_searching.cpp:26]: Thnakyou! I'll implement these changes

**Pull request closed without merging**